### ISSUE TITLE
Multiple users and servers

### ISSUE THREAD

**Issue by jmshrv** [jmshrv/finamp#10]
The user should be able to change what user/server they're using. I wouldn't count this as a "required" feature myself but doing this would massively change the way I'm currently storing data.

**Comment by redthing1** [comment]
How do you do multiple servers? I don't see a way.

**Comment by jmshrv** [comment]
idk why this is closed, I haven't added support yet. I may have closed it because I refactored storage to technically allow for it, but it hasn't actually been implemented yet.

**Comment by redthing1** [comment]
Oh great. Thanks.

**Comment by cyberwolfie1** [comment]
I would love this feature. Same use case as @magicDGS, where I have a server running locally which I would prefer to access via LAN when at home but ability to access through public URL when outside my home.

**Comment by bsfreq** [comment]
Yes! Having the ability to connect to two or more servers and enabling / disabling them and their independent music libraries would be a very nice addition. I remember reading this wasn't planned, but I'm happy to see this listed here. Hopefully someone can tackle this at some point.

**Comment by davpons-coder** [comment]
> This would be really useful.
> 
> Regarding remote/local access: I use PiHole and have added the server's remote domain as a local DNS entry with the relevant local IP address. This means it uses the local IP address locally and the remote IP address remotely.

Can you please explain with a bit more detail how to do this? I am just getting started and I am not very familiar with piHole.  I have just setup tailscale to access the library remotely but I don't want to switch servers between local and remote. I just did a standard install of piHole ...

**Comment by davpons-coder** [comment]
> > This would be really useful.
> > Regarding remote/local access: I use PiHole and have added the server's remote domain as a local DNS entry with the relevant local IP address. This means it uses the local IP address locally and the remote IP address remotely.
> 
> Can you please explain with a bit more detail how to do this? I am just getting started and I am not very familiar with piHole. I have just setup tailscale to access the library remotely but I don't want to switch servers between local and remote. I just did a standard install of piHole ...

Actually I figured it out...
I was confused because thought the process was to always use the local IP regardless of using local or remote access when in fact it is the opposite, you always use the remote domain.   

I Learned a bit so thanks a lot!!
